### PR TITLE
#20 added profile for cobertura-maven-plugin

### PR DESCRIPTION
**Pull request by asaen**

### CODE REVIEW

**Comment by karato** [comment]
Thanks for the pull request, let me find a reviewer..

**Comment by karato** [comment]
@alevohin review it

File: pom.xml [hunk 1]
```diff
@@ -185,6 +185,41 @@
...
+                        <artifactId>cobertura-maven-plugin</artifactId>
+                        <version>2.6</version>
+                        <configuration>
+                            <instrumentation>
+                                <excludes>
+                                </excludes>
+                            </instrumentation>
+                            <check>
+                                <haltOnFailure>true</haltOnFailure>
+                                <branchRate>0</branchRate>
```
Comment (alevohin) [inline pom.xml]: @darkled could we use something more then 0?

**Comment by alevohin** [comment]
@darkled please see one comment above.
And I suggest to replace `mvn clean install -Pqulice` with `mvn clean install -Pqulice,cobertura` in file `.travis.yml`, because Travis didn't invoke cobertura check now.

File: pom.xml (hunk 1 above)
Comment (darkled) [inline pom.xml]: @alevohin what value would you like to use?

**Comment by asaen** [comment]
@alevohin thanks, fixed

**Comment by asaen** [comment]
@alevohin finally, I reverted adding invocation of cobertura profile by travis since it is already a part of issue #34

**Comment by alevohin** [comment]
@darkled thank you!
@darkled @ggajos I suggest wait for #34 and merge master to this branch. Because we need to check that cobertura limits doen't broke master state, after merge we'll see Travis status.

**Comment by asaen** [comment]
@alevohin sure, we can, but I tried to commit it here as well and travis build passed successfully. So I do not expect any problems here.

**Comment by ggajos** [comment]
@alevohin up to you; if you think that it can violate build let's wait

**Comment by asaen** [comment]
@alevohin could you please take a look once more?

**Comment by alevohin** [comment]
@darkled thank you! Looks good!

**Comment by alevohin** [comment]
@rultor merge

**Comment by ggajos** [comment]
@rultor merge

**Comment by karato** [comment]
@alevohin **17 mins** sent to your balance (ID `52903522`), many thanks!; review comments (c=2) added as a bonus; added +17 to your rating, now it is equal to [+1271](http://www.netbout.com/b/35105?open=rating)

**Comment by karato** [comment]
@rultor deploy

**Pull request merged**